app/tests/base_test.py

```diff
@@ -69,7 +69,27 @@ def get_error_msg(code):
             )
             self.assertEqual(code, json_data['code'], msg="invalid response code")
 
-    def check_header_etag(self, etag, response):
+    def assertCacheControl(self, response, max_age=None, no_cache=False):  # pylint: disable=invalid-name
+        '''Assert that Cache-Control header is present and correct
+
+        Args:
+            response: HttpResponse
+                Response to check
+            max_age: int | None
+                Check that the header is `public, max-age=x`
+            no_cache: bool
+                Check that the header is `max-age=0, no-cache, no-store, must-revalidate, private`
+        '''
+        self.assertTrue(response.has_header('Cache-Control'), msg="Cache-Control header missing")
+        if no_cache:
+            self.assertEqual(
+                response['Cache-Control'],
+                'max-age=0, no-cache, no-store, must-revalidate, private'
+            )
+        elif max_age is not None:
+            self.assertEqual(response['Cache-Control'], f'max-age={max_age}, public')
```

Comment:
Probably add a check, that `max_age` is less-equal the max allowed cache duration (1h)?

Reply:
@hansmannj See comment above, the maximum value or 1h is just because the current algorithm is not optimal for bigger value, but in theory and in future we should not have any limitation. Therefore I don't it worth the effort to add a check here need to be modified/removed in future.

Reply:
See my comment above. But we can leave it as it is.